wrf-cmake.rb

```diff
@@ -0,0 +1,77 @@
+# WRF-CMake (https://github.com/WRF-CMake/WRF).
+# Copyright 2019 M. Riechert and D. Meyer. Licensed under the MIT License.
+
+class WrfCmake < Formula
+  desc "The Weather Research and Forecasting (WRF) model with CMake support"
+  homepage "https://github.com/WRF-CMake/WRF"
+  url "https://github.com/WRF-CMake/WRF.git",
+      :branch => "wrf-cmake",
+      :commit => "8b9edc944f0cbbfb22966850ae3724f617f0f705"
+  version "4.1.wrf-cmake"
+  sha256 "37814ee7bfe7077cba8bd0175258ef763380fce3e7d8aab2ea8360a902a11043"
+  head "https://github.com/WRF-CMake/WRF.git", :branch => "wrf-cmake"
+
+  depends_on "cmake" => :build
+  depends_on "gcc" # for gfortran
+  depends_on "jasper"
+  depends_on "libpng"
+  depends_on "netcdf"
+  depends_on "open-mpi"
+
+  resource "WPS" do
+    url "https://github.com/WRF-CMake/WPS.git",
+        :branch => "wps-cmake",
+        :commit => "d918dc6da17b177aa1fe0082866274968f45a171"
+  end
+
+  def install
+    (buildpath/"WPS").install resource("WPS")
+
+    # Prevent CMake from finding & using ifort
+    ENV["FC"] = Formula["gcc"].opt_bin/"gfortran"
+
+    wrf_args = *std_cmake_args.map + %W[
```

Comment:
Ha, one more thing! :D
Here you use `* ... .map` but for WPS it's without those. Which one is better? Whatever it is, best to make it consistent.

Reply:
That would be known as a PEBCAK error. Also known as an ID10T error, or copy pasta error. I'll push a fix.

Do you want me to squash my commits? I guess it doesn't matter, you could do it through github if it so pleases you. If you want me to, please just say the word.

Reply:
I will actually keep *all* your commits. They're beautifully commented and all make sense. I'm usually too lazy and typically slip in a few "fix", "wip", "foo" commits and just squash everything at the end. But yes, your history deserves to be archived.